.github/workflows/test_and_deploy.yml

```diff
@@ -98,12 +104,11 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -U setuptools setuptools_scm wheel twine
-      - name: Build and publish
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.TWINE_API_KEY }}
+          pip install -U setuptools setuptools_scm wheel
+      - name: Build python package
         run: |
           git tag
           python setup.py sdist bdist_wheel
```

Comment:
Also, do you want to switch to using `python -m build` too?

Reply:
Yes, please do this